app/controllers/application_controller.rb

```diff
@@ -31,7 +31,7 @@ def require_admin
   end
 
   def authorize!
-    permission = PermissionService.new(current_user, params[:store_slug], params[:controller], params[:action])
+    permission = PermissionService.new(current_user, params[:controller], params[:action])
```

Comment:
wow great job aram!

Reply:
awesome - I'm happier with that slug outta there